From e6e8c162d2e6a078a330c2b46b2ee9ddcd53b754 Mon Sep 17 00:00:00 2001
From: Sandipan Das <sandipan.das@amd.com>
Date: Mon, 3 Jun 2024 13:44:54 +0530
Subject: [PATCH 05/32] readme: Add note on AMD support

Call out support for AMD EPYC processors from the 7001, 7002, 7003,
4004, 8004 and 9004 series.

Signed-off-by: Sandipan Das <sandipan.das@amd.com>
---
 README.md | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/README.md b/README.md
index 9908e92..fc2db9e 100644
--- a/README.md
+++ b/README.md
@@ -77,6 +77,9 @@ numatop is supported on Intel Xeon processors: 5500-series, 6500/7500-series,
 E5-16xx/24xx/26xx/46xx-series had better be updated to latest CPU microcode
 (microcode must be 0x618+ or 0x70c+).
 
+AMD EPYC processors from the 7001, 7002, 7003, 4004, 8004 and 9004 series are
+also supported.
+
 To learn about NumaTOP, please visit http://01.org/numatop
 
 
-- 
2.41.0

